packages/staking/src/features/browse-pools/StakePoolsTable/StakePoolsTable.module.scss

```
@@ -1,28 +1,10 @@
 @import '../../../../../common/src/ui/styles/theme.scss';
 @import '../../../../../common/src/ui/styles/abstracts/typography';
 
-.table {
-  display: flex;
-  flex-direction: column;
-  padding: size_unit(4) 0;
-  min-height: 990px;
-  .header {
+.stakePoolTable {
```

naming nitpick for the sake of consistency
```suggestion
.stakePoolsTable {
```

---

Fixed on d5ccbf4